From f3e5b902431a89c210005b20dafe909d180dd9c3 Mon Sep 17 00:00:00 2001
From: Marcelo Vanzin <vanzin@cloudera.com>
Date: Fri, 9 Oct 2015 11:33:04 -0700
Subject: [PATCH 017/201] CLOUDERA-BUILD. CDH-31874. Remove scripts that don't
 work from package.

CDH doesn't include the code that these scripts need, so they just don't
work.

(cherry picked from commit 52aff1dea7e3e6fef3793a78a68fbe3a50e47a6f)
---
 make-distribution.sh |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/make-distribution.sh b/make-distribution.sh
index 6025e3e..fd9126f 100755
--- a/make-distribution.sh
+++ b/make-distribution.sh
@@ -228,6 +228,13 @@ if [ -d "$SPARK_HOME"/R/lib/SparkR ]; then
   cp "$SPARK_HOME/R/lib/sparkr.zip" "$DISTDIR"/R/lib
 fi
 
+# CDH: remove scripts for which the actual code is not included.
+rm "$DISTDIR/bin/spark-sql"
+rm "$DISTDIR/bin/beeline"
+rm "$DISTDIR/bin/sparkR"
+rm "$DISTDIR/sbin/start-thriftserver.sh"
+rm "$DISTDIR/sbin/stop-thriftserver.sh"
+
 # Download and copy in tachyon, if requested
 if [ "$SPARK_TACHYON" == "true" ]; then
   TMPD=`mktemp -d 2>/dev/null || mktemp -d -t 'disttmp'`
-- 
1.7.9.5

